### PR TITLE
Add UX working group notes for 01-29-2021

### PR DESCRIPTION
**Pull request by michdolan**
I tried to organize the points raised by everyone, but not sure I captured everything. Please have a look and let me know if you have edits @doug-walker @chrisbrejon @zachlewis @carolalynn22 and Mark Titchener (not sure Mark's GH handle).

Signed-off-by: Michael Dolan <michdolan@gmail.com>

### CODE REVIEW

File: ASWF/meetings/working_groups/ux/2021-01-29.md [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+      space values, etc. Helpful to clarify intent of roles, provide use cases, 
+      user stories. What could improve usability.
+
+* Roles:
+    - Mark: Roles have been fundamentally misunderstood, how they were 
+      intended to be implemented in DCC. With roles in Nuke, did what made 
+      sense for each input file type. Can we come up more consistent roles 
+      between different applications? Trying to keep those available more 
+      consistently used, and then users can extend as needed. Mari needed more 
+      roles than were available, and wanted to get asway from confusion of
```
Comment (EvanAW) [inline ASWF/meetings/working_groups/ux/2021-01-29.md:37]: Minor spelling nitpick: asway->away
```suggestion
      roles than were available, and wanted to get away from confusion of 
```

**Comment by chrisbrejon** [comment]
Looks great to me ! Thanks Michael !

File: ASWF/meetings/working_groups/ux/2021-01-29.md (hunk 1 above)
Comment (michdolan) [reply ASWF/meetings/working_groups/ux/2021-01-29.md:37]: Thanks.

File: ASWF/meetings/working_groups/ux/2021-01-29.md [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+
+Attendees:
+  * [X] Michael Dolan (_TSC Chair_) - Epic Games
+  * [X] Carol Payne (_TSC_) - Netflix
+  * [X] Mark Titchener (_TSC_) - Foundry
+  * [X] Doug Walker (_TSC Chief Architect_) - Autodesk
+  * [X] Christophe Brejon
+  * [X] Zach Lewis - Method
+
+# **OCIO Configs Working Group Meeting Notes**
```
Comment (EvanAW) [inline ASWF/meetings/working_groups/ux/2021-01-29.md:16]: OCIO Configs Working Group Meeting Notes
```suggestion
# **OCIO UX Working Group Meeting Notes**
```

File: ASWF/meetings/working_groups/ux/2021-01-29.md [hunk 3]
```diff
@@ -0,0 +1,154 @@
...
+      guidance on what to do.
+    - Zach: Color space attributes can inform that.
+
+* Dynamic parameters:
+    - Michael: Dynamic parameters example for supporting uniforms would be 
+      good. Can be a little confusing at first.
+    - Doug: Unit tests best source right now.
+
+* Grading transforms:
+    - Zach: Good to have beasic interface for playing with RGB grading curves
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/ux/2021-01-29.md:101]:
```suggestion
    - Zach: Good to have basic interface for playing with RGB grading curves 
```

File: ASWF/meetings/working_groups/ux/2021-01-29.md [hunk 4]
```diff
@@ -0,0 +1,154 @@
...
+      AD product docs, but doesnt include all of it. To clarify on CTF. Started 
+      as AD thing. Now think of it as native format of OCIO. Can always 
+      serailize to this format.
+
+* Task distribution:
+    - Michael: For this meeting: Topical discussion on different pieces? One 
+      per meeting perhaps.
+    - Doug: Yeah, how do you make progress in wg? Hard to have tangible output.
+    - Carol: This group similar to ACES implementation groups. If implementors 
+      not part of conversation, not sure. If works if software implementors can
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/ux/2021-01-29.md:117]:
```suggestion
      not part of conversation, not sure. It works if software implementors can 
```

File: ASWF/meetings/working_groups/ux/2021-01-29.md [hunk 5]
```diff
@@ -0,0 +1,154 @@
...
+      serailize to this format.
+
+* Task distribution:
+    - Michael: For this meeting: Topical discussion on different pieces? One 
+      per meeting perhaps.
+    - Doug: Yeah, how do you make progress in wg? Hard to have tangible output.
+    - Carol: This group similar to ACES implementation groups. If implementors 
+      not part of conversation, not sure. If works if software implementors can 
+      come to an agreement.
+    - Doug: How are going to make progress in group? Comes down to what extent
```
Comment (Simran-B) [inline ASWF/meetings/working_groups/ux/2021-01-29.md:119]:
```suggestion
    - Doug: How are we going to make progress in group? Comes down to what extent 
```

**Review by doug-walker — APPROVED** [review]

File: ASWF/meetings/working_groups/ux/2021-01-29.md (hunk 4 above)
Comment (michdolan) [reply ASWF/meetings/working_groups/ux/2021-01-29.md]: Fixed.

File: ASWF/meetings/working_groups/ux/2021-01-29.md (hunk 3 above)
Comment (michdolan) [reply ASWF/meetings/working_groups/ux/2021-01-29.md]: Fixed

File: ASWF/meetings/working_groups/ux/2021-01-29.md (hunk 2 above)
Comment (michdolan) [reply ASWF/meetings/working_groups/ux/2021-01-29.md]: Fixed

**Comment by michdolan** [comment]
Closing PR since these notes have been moved to Confluence: https://wiki.aswf.io/display/OCIO/2021-01-29

**Pull request closed without merging**